site/content/recipes/perform_code_review.md

```diff
@@ -37,14 +37,35 @@
 
 ## Reviewer reviews code
 
-1. Reviewer reminds themselves of objectives and areas of focus from first meeting (link to Meet and Agree on Objectives section)
+What should you do if you're the reviewer, providing feedback on someone else's
+code?
+We suggest a workflow with two key elements.
+First of all, keep the authors' objectives in mind.
+To achieve this, begin your review by refreshing your memory about what you and
+the author discussed when you met initially.
+The second key element is to have a checklist.
+We provide such a checklist here:
+https://github.com/OxfordCodeReviewNet/forum/blob/master/guidelines_for_reviewers.md.
+Of course, no checklist can perfectly capture all the dimensions of research
+code review.
+Our goal in providing a checklist is to give beginners somewhere to start, and
```

Comment:
yes, I think that's a fundamental point we want to make here. Often beginners lack the confidence to review because they don't see what they could possibly say. But with a checklist or clear guidelines, they can start with confidence.